src/mapref/multiple.rs

```diff
@@ -94,11 +94,11 @@ impl<'a, K: Eq + Hash, V, S: BuildHasher> RefMutMulti<'a, K, V, S> {
         self.v
     }
 
-    pub fn pair(&self) -> (&K, &V) {
+    pub fn pair(&self) -> (&'a K, &V) {
```

Comment:
The value here (and in a couple other locations) needs a lifetime annotation as well.

Reply:
I don't think it is possible to extend the lifetimes for the remaining values, since the `RefMut*` structs contain mutable references to the values.